content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md

```diff
@@ -100,10 +100,12 @@ community-owned repositories (`pkgs.k8s.io`).
 
 ## Can I continue to use the legacy package repositories?
 
-The existing packages in the legacy repositories will be available for the foreseeable
+~~The existing packages in the legacy repositories will be available for the foreseeable
 future. However, the Kubernetes project can't provide _any_ guarantees on how long
 is that going to be. The deprecated legacy repositories, and their contents, might
-be removed at any time in the future and without a further notice period.
+be removed at any time in the future and without a further notice period.~~
+
+**UPDATE: The legacy packages are expected to go away in January 2024.**
```

Comment:
(nit)
```suggestion
**UPDATE**: The legacy packages are expected to go away in January 2024.
```

It'd be nice to make a shortcode for this kind of thing.

Reply:
done